android/java/org/chromium/chrome/browser/app/domain/NetworkModel.java

```diff
@@ -43,7 +43,6 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import java.util.Set;
 import java.util.stream.Collectors;
 
 public class NetworkModel implements JsonRpcServiceObserver {
```

Comment:
`getSubTestNetworks` method is unused and can be safely removed now.

Reply:
Updated https://github.com/brave/brave-core/pull/19467#discussion_r1278958347